frontend/src/routes/saves/$saveId.tsx

```
@@ -0,0 +1,13 @@
+import { useQuery } from "@tanstack/react-query";
+import { createFileRoute } from "@tanstack/react-router";
+
+export const Route = createFileRoute("/saves/$saveId")({
+  component: RouteComponent,
+});
+
+function RouteComponent() {
+  const {} = useQuery({
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)